### ISSUE TITLE
ID 256175: [Supporting the platform -Simon – Search Quick Ranges]: Contents present under “Absolute time range” blade disappear after resizing the page to 200% Zoom.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66363]
### **User Experience:**  
Users who have low vision will be affected if controls disappear after increasing the screen zoom to 200% as they will not be able to access the controls present on the screen.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials.  "Home" page appears. 
2. Navigate the page. 
3. Navigate the header controls "Apply time range”, a new pane appears. 
4. Navigate to the"Search Quick Ranges" search edit field. 
5. Search time ranges and observe. 
6. Verify and observe the issue.

**Actual Result:** 
Issue:  Contents present under “Absolute time range” blade are disappearing afterresizing the page to 200%.
Observation:
While resizing the page to 200% controls (Changetime settings), links (Read the Documentation) and description (It look like it haven’t used) disappeared.  

**Expected Result:**
Solution: When resizing the page all the contents should be visible and should not get disappeared.
_Note: The same issue exists under "Explore" screen. (Home page>Explore) Same issue appears for Explore->Inspector._

### **Severity:**
`SEV2`

### **Area:**
`Time Range Picker` (Saga Design System)

**Comment by joshhunt** [comment]
When at large zoom values, the time range picker gets clipped by the window and does not scroll fully.

![image](https://user-images.githubusercontent.com/46142/232004755-708a6782-2235-4bbe-a88e-609ef3436a8e.png)

**Comment by ashharrison90** [comment]
this is still an issue

**Comment by ashharrison90** [comment]
related escalation: https://github.com/grafana/support-escalations/issues/14406
